sphinx_doc/setup_guide.rst

```diff
@@ -1,328 +1,246 @@
-    Python 2.7.8  
-
+    sudo make altinstall
+    
+    # Update PATH:
     export PATH="/usr/local/bin:$PATH"
-
-Intall pip and virtualenv::
+    
+    # Checking Python version (output should be: Python 2.7.8):
+    python2.7 -V
+    
+Install pip and virtualenv::
 
     wget https://bootstrap.pypa.io/ez_setup.py
-    python2.7 ez_setup.py
+    sudo /usr/local/bin/python2.7 ez_setup.py
+    
     wget https://bootstrap.pypa.io/get-pip.py
-    python2.7 get-pip.py
-    pip2.7 install virtualenv
-
-Build a separated virtualenv::
+    sudo /usr/local/bin/python2.7 get-pip.py
+    
+    sudo /usr/local/bin/pip2.7 install virtualenv
 
-    # create a virtual environment called py2.7. 
-    # Activate it by source py2.7/bin/activat
+Build a separate virtualenv::
 
-    # Make dir '/path/to/i5k/virtualenv' and switch in
-    mkdir /path/to/i5k/virtualenv 
-    cm /path/to/i5k/virtualenv
-    /usr/bin/virtualenv py2.7 
+    # Make root dir for virtualenv and cd into it:
+    mkdir <virt-env>
+    cd <virt-env> 
+    
+    # Create a virtual environment called py2.7 and activate:
+    virtualenv py2.7 
     source py2.7/bin/activate
-
-RabbitMQ 
-~~~~~~~~
+    
+    # Install the project in the virtual environment:
+    cd <app-home> 
+    cp -pr <app-home> <virt-env> 
+    
+Python Modules and Packages
+---------------------------
+
+Install additional Python packages::
+
+    cd <virt-env>
+     
+    # Cut, paste and run the following bash script.
+    # If any installation fails script halts:  
+    for package in                         \
+        "django==1.8.12"                   \
+        "markdown==2.6.6"                  \
+        "cssmin==0.2.0"                    \
+        "django-pipeline==1.6.8"           \
+        "djangorestframework==2.3.4"       \
+        "django-rest-swagger==0.3.5"       \
+        "django-suit==0.2.18"              \
+        "django-axes"                      \
+        "docutils==0.12"                   \
+        "jsmin==2.0.11"                    \
+        "pycrypto==2.6.1"                  \
+        "python-memcached==1.57"           \
+        "python-social-auth==0.2.16"       \
+        "requests-oauthlib==0.6.1"         \
+        "wsgiref==0.1.2"                   \
+        "pillow==2.2.2"                    \
+        "django-simple-captcha==0.4.5"
+    do
+        echo -e "\nInstalling $package..."
+        if ! yes | pip install $package ; then 
+            echo -e "\nInstallation of package $package FAILED"
+            break
+        fi
+    done
+    
+RabbitMQ
+--------
 
 Install RabbitMQ Server::
 
-    ## RHEL/CentOS 7 64-Bit ##
-    wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
-    rpm -ivh epel-release-7-5.noarch.rpm
+    # Install RHEL/CentOS 7 64-Bit Extra Packages for Enterprise Linux (Epel): 
+    wget https://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+    sudo rpm -ivh epel-release-7-5.noarch.rpm
```

Comment:
On 06/29/2016 01:58 PM, childers wrote:
>
> In sphinx_doc/setup_guide.rst 
> <https://github.com/NAL-i5K/django-blast/pull/2#discussion_r68996114>:
>
> >
> >  Install RabbitMQ Server::
> >
> > -    ## RHEL/CentOS 7 64-Bit ##
> > -    wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
> > -    rpm -ivh epel-release-7-5.noarch.rpm
> > +    # Install RHEL/CentOS 7 64-Bit Extra Packages for Enterprise Linux (Epel):
> > +    wget https://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
> > +    sudo rpm -ivh epel-release-7-5.noarch.rpm
>
> No worries, that's what collaborative editing is all about, right?
>

Thanks.  And than God for fresh pairs of eyes...  8-)

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/NAL-i5K/django-blast/pull/2/files/6fb9f04451a3b598752457f2347e5bc6554e40e0#r68996114>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ATKh81arXgoV4KNSdrAA6eBJUiVk3ilBks5qQrJCgaJpZM4JAUXb>.
>